esmvaltool/namelist_schema.yml

```diff
@@ -0,0 +1,39 @@
+# Generic namelist Yamale schema definition.
+# See https://github.com/23andMe/Yamale for help.
+
+---
+# Namelist schema
+models: list(include('model'), required=False)
+preprocessors: map(map(), min=1)
+diagnostics: map(include('diagnostic'), required=False)
+
+---
+# Namelist item definitions
+model:
+  model: str()
+  start_year: int(min=1750, max=2250)
+  end_year: int(min=1750, max=2250)
+  ensemble: str(required=False)
+  exp: str(required=False)
```

Comment:
This is required.

Reply:
Some 'models' don't have this facet, those with project = OBS for example.

Reply:
Correct. I was confusing `model` with `exp`...